app/models/airline.rb

```diff
@@ -7,4 +7,15 @@ class Airline < ActiveRecord::Base
   def self.search(query)
     where("name ilike :q or description ilike :q", q: "%#{query}%")
   end
+
+  def average_rating
+    sum = self.reviews.all.inject(0) { |sum, review| sum += review.rating }
```

Comment:
Shadowing outer local variable - `sum`.<br>Useless assignment to variable - `sum`. Use just operator `+`.<br>Redundant `self` detected.